### PR TITLE
feat: Open skills Activity by swapping from right to left

### LINKED ISSUE #1924: Swipe to go to susi-skills
**Describe the feature you'd like**

Swipe left in order to go to susi-skills and vice versa to come back. 
The user need not always go and touch the susi icon in order to go to the susi skills.
rather he can swipe left .

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #1924 

Changes: 
Refactored the code. Used  the concept of innerclass

Screenshot:
qwerty

### CODE REVIEW

**Comment by atm1504** [comment]
@iamareebjamal please review it.

File: app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt [hunk 1]
```diff
@@ -101,6 +106,25 @@ class ChatActivity : AppCompatActivity(), IChatView {
...
+    }
+    internal inner class CustomGestureListener : GestureDetector.SimpleOnGestureListener() {
+
+        override fun onFling(
+            event1: MotionEvent,
+            event2: MotionEvent,
+            velocityX: Float,
+            velocityY: Float
+        ): Boolean {
+            if (event2.x < event1.x) {
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: This will require very small gesture

Comment (atm1504) [reply app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: Sir, shall I use manual values like 
Screenshot 2019-03-30 at 2 01 43 AM
this to handle gesture.

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt]: Yes, I'm not an expert on gesture but a nudge should not open the activity

**Comment by atm1504** [comment]
Sir, I have updated the code. You can check it. Now it does not detect minor gestures. It only works when the gesture is properly, i.e, made intentionally. I have tasted it by trying various possibilities

**Pull request merged**